services/supervisor/src/lib.rs

```diff
@@ -71,24 +71,27 @@ pub use self::{
     errors::Error,
     proto_structures::{
         ConfigChange, ConfigProposalWithHash, ConfigPropose, ConfigVote, DeployConfirmation,
-        DeployRequest, ServiceConfig, StartService,
+        DeployRequest, ServiceConfig, StartService, SupervisorConfig,
     },
     schema::Schema,
     transactions::SupervisorInterface,
 };
 
 use exonum::{
+    blockchain::config::InstanceInitParams,
     crypto::Hash,
     runtime::{
         rust::{
-            api::ServiceApiBuilder, AfterCommitContext, Broadcaster, CallContext, DefaultInstance,
-            Service,
+            api::ServiceApiBuilder, AfterCommitContext, Broadcaster, CallContext, Service,
+            ServiceFactory as ServiceFactoryTrait,
```

Comment:
Nit: Shouldn't it be possible to use `ServiceFactory as _`? Or was this introduced after Rust 1.36?